core/src/main/java/edu/wpi/grip/core/operations/Operations.java

```diff
@@ -15,6 +15,7 @@ private Operations() { /* no op */}
 
     public static void addOperations(EventBus eventBus) {
         // Add the default built-in operations to the palette
+        eventBus.post(new OperationAddedEvent(new ResizeOperation()));
```

Comment:
To rephrase @AustinShalit's question.
We should have some sort of policy for ordering and defining how we want the palette ordered.
Why we are putting specific operations in a specified order.

Do we have a policy like this? Can we come up with one?